src/libraries/System.Text.Json/tests/System.Text.Json.Tests/JsonDocumentTests.cs

```diff
@@ -2047,6 +2047,42 @@ void assertOdd(JsonElement elem)
             }
         }
 
+        public static IEnumerable<object[]> InvalidDateTimePropertyValueData()
+        {
+            foreach (var str in new string[] { ">", "\u003e" })
+            {
+                // DateTime value length less than System.Text.Json.JsonConstants.MinimumDateTimeParseLength = 10
+                // DateTime value length more than System.Text.Json.JsonConstants.MaximumEscapedDateTimeOffsetParseLength = 252
+                foreach (var count in new int[] { 9, 253 })
+                {
+                    yield return new object[] { string.Join("", Enumerable.Repeat(str, count)) };
+                }
+            }
+        }
+
+
+        [Theory]
```

Comment:
While your changes do address the bug as originally reported, these tests are not validating that this is the case: I tried running them without your fix and they're still passing. Please add tests verifying that bug as originally reporting has been fixed.

Reply:
My bad. I just kept thinking about the fact that the DateTime length should be between 10 and 252 and missed the real test case. I have added it now and removed the <10 and >252 test cases as they anyways succeed in any case.